cpp/memilio/timer/definitions.cpp

```diff
@@ -0,0 +1,72 @@
+/* 
+* Copyright (C) 2020-2025 MEmilio
+*
+* Authors: Rene Schmieding
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#include "memilio/timer/definitions.h"
+
+namespace mio
+{
+namespace timing
+{
+
+#ifdef MEMILIO_ENABLE_OPENMP
+
+/**
+ * @brief Convert a duration to a (floating point) number of seconds.
+ * @param[in] duration Any DurationType value, mainly `BasicTimer::get_elapsed_time()`.  
+ * @return The duration in seconds.
+ */
+double time_in_seconds(DurationType duration)
+{
+    return duration;
+}
+
+/**
+ * @brief Get the current time.
+ * @return Returns omp_get_wtime() if OpenMP is enabled, stead_clock::now() otherwise.
+ */
+TimeType get_time_now()
+{
+    return omp_get_wtime();
+}
+
+#else
+
+/**
+ * @brief Convert a duration to a (floating point) number of seconds.
+ * @param[in] duration Any DurationType value, mainly `BasicTimer::get_elapsed_time()`.  
+ * @return The duration in seconds.
+ */
+double time_in_seconds(DurationType duration)
+{
+    return std::chrono::duration_cast<std::chrono::duration<double>>(duration).count();
+}
+
+/**
+ * @brief Get the current time.
+ * @return Returns omp_get_wtime() if OpenMP is enabled, stead_clock::now() otherwise.
```

Comment:
```suggestion
 * @return Returns omp_get_wtime() if OpenMP is enabled, steady_clock::now() otherwise.
```

Reply:
Well spotted. I will remove documentation from the .cpp, and only fix this in the .h file.